### ISSUE TITLE
got IndexError

### ISSUE THREAD

**Issue by DOQQLOL** [NVIDIA/retinanet-examples#309]
boxes = box_head[a, :, y, x]
IndexError: tensors used as indices must be long, byte or bool tensors

https://github.com/NVIDIA/retinanet-examples/blob/main/odtk/box.py#L299

i wonder why not use '/'  instead of  '//'  to get the index?

i changed the code to 
`        # classes = indices // width // height % num_classes  # modified
        # classes = classes.type(all_cls_head.type())
        #
        # # Infer kept bboxes
        # x = indices % width  # torch.int64
        # y = indices // width % height  # modified
        # a = indices // width // height // num_classes  # modified`

**Comment by ankushgupta7** [comment]
I agree, I faced the same problem and it makes sense to modify the code as you have documented above. Single '/' in python can give float values, but '//' will give the integer value. After the modification, It's working fine.
